src/sentry/api/endpoints/organization_releases.py

```diff
@@ -623,13 +624,14 @@ def post(self, request: Request, organization) -> Response:
                 status = 201
 
             analytics.record(
-                "release.created",
-                user_id=request.user.id if request.user and request.user.id else None,
-                organization_id=organization.id,
-                project_ids=[project.id for project in projects],
-                user_agent=request.META.get("HTTP_USER_AGENT", ""),
-                created_status=status,
-                auth_type=get_auth_api_token_type(request.auth),
+                ReleaseCreatedEvent(
+                    user_id=request.user.id if request.user and request.user.id else None,
```

Comment:
I don't think the explicit check for `request.user.id`'s truthiness is needed here, but it's also not harmful (and I'm now realizing it is probably out of scope to fix in this PR)

```suggestion
                    user_id=request.user.id if request.user else None,
```

Reply:
I think `request.user.id` can be `None` when the model is not yet saved. I think the `and`-clause is probably overkill.

Reply:
FWIW I wouldn't touch the logic here - if it was previously done this way, let's keep it. We're not trying to change behaviour here.